Paginated history view

shaneboulden opened the pull request:
I've added a basic paginated history view; pics are in the Trello ticket. I've already created a PR for the `peakorc` changes required to enable pagination for suites data. I'll leave this as a WIP until I've figured out the JavaScript required to update the number of available pages, and highlight the active page.

shaneboulden committed:
render history view

shaneboulden committed:
add history view

shaneboulden committed:
conditionally disable page controls

shaneboulden commented:
Updated the template and route to include total pages. The history view now includes controls for the number of pages returned by `peakorc`, and conditionally disables the 'next' and 'previous' controls.

adamgoossens reviewed (commented):
Is it worth having more explicit URL injection protection, or is the typecast to int() going to be sufficient? I suspect the typecast will be fine, but it'll throw a nasty exception. A dedicated check would allow us to catch any nastiness and give a meaningful error message.

LGTM.

shaneboulden committed:
handle error when page var is non-integer

shaneboulden commented:
Great pickup! You're absolutely right - that exception was nasty, the message concerned not being able to decode a JSON field...

I've just added a 500 template and added a commit to handle the exception when a non-integer value is specified. There's a screenshot below of the resulting 500 page.

![screenshot from 2018-01-28 22-15-02](https://user-images.githubusercontent.com/1834999/35481544-7f617888-0471-11e8-8593-565c4a34b704.png)
